Jantimizer/Histograms/HistogramModels/HistogramEquiDepth.cs

```diff
@@ -38,9 +38,13 @@ private void GenerateHistogramFromSorted(List<int> sorted)
         {
             for (int bStart = 0; bStart < sorted.Count; bStart += Depth)
             {
-                IHistogramBucket newBucket = new HistogramBucket(sorted[bStart], 1);
-                for (int bIter = bStart + 1; bIter < bStart + Depth && bIter < sorted.Count; bIter++)
+                IHistogramBucket newBucket = new HistogramBucket(sorted[bStart], sorted[bStart], 1);
+
+                for (int bIter = bStart + 1; bIter < bStart + Depth && bIter < sorted.Count; bIter++) {
                     newBucket.Count++;
+                    if (sorted[bIter] > newBucket.ValueEnd)
```

Comment:
As `sorted` is sorted, `if (sorted[bIter] > newBucket.ValueEnd)` will always be true